target-bin/bootstrap-fixup.in

```diff
@@ -16,13 +16,15 @@ elif [ $DISTRIB_NAME = "Debian" ]; then
   # grub-legacy conflicts grub-pc dependencies
   # No grub-legacy on Ubuntu, just on Debian
   # Work around bcron-run conflict due to cron being removed
-  apt-get purge -y grub-legacy bcron-run &> /dev/null
+  # Needed for KVM, but apparently errors out for LXC, so the true ignores the error
+  apt-get purge -y grub-legacy bcron-run &> /dev/null || true
 fi
 echo '127.0.1.1 gitian' >> /etc/hosts
 
 # If LXC
 if grep /lxc/gitian /proc/1/cgroup > /dev/null; then
-  adduser --disabled-password --gecos ubuntu --quiet ubuntu || true
+  adduser --disabled-password --gecos ${DISTRIB_NAME,,} --quiet ${DISTRIB_NAME,,} || true
+  adduser ${DISTRIB_NAME,,} sudo
```

Comment:
Looking at this more closely, it does seem odd. The Debian part of vmbuilder functions the same as the Ubuntu part in this respect in that it creates an admin group, adds the user to the admin group and installs the sudoers file from a template that allows sudo privileges for the admin group.

But maybe @boklm can comment on the issues that the Tor Project has that requires that the user be added to sudo specifically, since I got this from one of the commits he linked.